web/packages/teleport/src/AuthConnectors/AuthConnectorTile.tsx

```diff
@@ -0,0 +1,227 @@
+            )}
+          </Flex>
+          <Subtitle2 color="text.slightlyMuted">{customDesc || desc}</Subtitle2>
+        </Flex>
+      </Flex>
+      <Flex
+        flexDirection="column"
+        justifyContent="space-between"
+        alignItems="flex-end"
+        height="100%"
+      >
+        <ActionsSection />
+        {!!onEdit && !!onDelete && (
+          <MenuIcon
+            buttonIconProps={{ style: { borderRadius: `${theme.radii[2]}px` } }}
+          >
+            <MenuItem onClick={onClickEdit}>Edit</MenuItem>
+            <MenuItem onClick={onClickDelete}>Delete</MenuItem>
+          </MenuIcon>
+        )}
+      </Flex>
+    </ConnectorBox>
+  );
+}
+
+/**
+ * LocalConnectorTile is a hardcoded "auth connector" which represents local auth.
+ */
+export function LocalConnectorTile() {
+  return (
+    <AuthConnectorTile
+      key="local-auth-tile"
+      kind="local"
+      id="local"
+      Icon={() => (
+        <Flex
+          alignItems="center"
+          justifyContent="center"
+          css={`
+            background-color: ${props =>
+              props.theme.colors.interactive.tonal.neutral[0]};
+            height: 61px;
+            width: 61px;
+          `}
+          lineHeight={0}
+          p={2}
+          borderRadius={3}
+        >
+          <Password size="extra-large" />
+        </Flex>
+      )}
+      isDefault={true}
+      isPlaceholder={false}
+      isEnabled={true}
+      name="Local Connector"
+      customDesc="Manual auth w/ users local to Teleport"
+    />
+  );
+}
+
+function EnabledIndicator({ isEnabled }: { isEnabled: boolean }) {
+  return (
+    <Flex
+      justifyContent="center"
+      alignItems="center"
+      gap={1}
+      py={1}
+      px={2}
+      css={`
+        background-color: ${isEnabled
+          ? props => props.theme.colors.interactive.tonal.success[0]
+          : props => props.theme.colors.interactive.tonal.neutral[0]};
+        width: 85px;
+        height: 24px;
+        border-radius: 62px;
+      `}
+    >
+      {isEnabled ? (
+        <>
+          <CircleCheck color="interactive.solid.success.default" size="small" />
+          <P3 color="interactive.solid.success.default">Enabled</P3>
+        </>
+      ) : (
+        <>
+          <MinusCircle color="text.disabled" size="small" />
+          <P3 color="text.disabled">Disabled</P3>
+        </>
+      )}
+    </Flex>
+  );
+}
+
+type Props = {
+  name: string;
+  id: string;
+  kind: AuthType;
+  Icon: () => JSX.Element;
+  isDefault: boolean;
+  /**
+   * isPlaceholder is whether this isn't a real existing connector, but a placeholder as a shortcut to set one up.
+   */
+  isPlaceholder: boolean;
+  onSetup?: () => void;
+  customDesc?: string;
+  isEnabled: boolean;
+  onEdit?: ResourceState['edit'];
+  onDelete?: ResourceState['remove'];
+  /**
+   * isCTA is whether this tile isn't a real existing conncetor, but a CTA (call to action) to upgrade to Enterprise.
+   */
+  isCTA?: boolean;
```

Comment:
Can something be a placeholder and a CTA at the same time? If not, then consider replacing `isPlaceholder` and `isCTA` with something like `kind: 'regular' | 'placeholder' | 'cta'`.